scripts/how_to/adding_new_ida_to_toggle_report.rst

```diff
@@ -0,0 +1,135 @@
+=====================================
+Enabling Toggles Report for a new IDA
+=====================================
+
+The generation of toggles report for an IDA has been automated. However, for now, the automation has only been used on lms. This document lists what needs to be done to automate toggle report generation for other IDAs.
+
+.. contents:: Steps
+
+Gathering Data
+==============
+
+Annotation Data
+---------------
+The toggles are should be annotated in code following edX code annotation:`writing annotations`_. The `code_annotations`_ tool is used to collect annotations into a report.
+
+The steps to collect annotations is automated through a jenkins job
+  - `groovy job specification`_ in "generate-code-annotation-report"
+  - `Jenkins Job folder`_
+  - once the job is done, its data is pushed to s3 bucket: `script to push data to s3 bucket`_
+
+
+.. _writing anntations: https://code-annotations.readthedocs.io/en/latest/writing_annotations.html
+.. _code_annotations: https://www.github.com/edx/code-annotations
+
+steps
+~~~~~
+* add a `generate-feature-toggle-annotation-report.sh`_ to repository, use linked file as example
+* checkout your IDA code repository in groovy using git block
```

Comment:
We could make this less manual by having a dict in the groovy file that we loop over to add `git` and `virtualenv` blocks to the jobs.

Reply:
Oh, I see this is mentioned below. Maybe we can leave the how-to this way for now and then make this change right after.

Reply:
Yeah, I tried to capture what would need to happen now to add an IDA. The plan is to simplify the steps in a future ticket and update this how_to

Reply:
OK cool, so this is just a snapshot of the process as it stands now, not what we want it to be.